src/commands/logout.rs

```diff
@@ -41,6 +41,8 @@ impl LogoutCommand {
     pub async fn exec(self) -> Result<()> {
         self.keyring_entry
             .delete_entry(self.common.read_config()?.home_url)?;
+        let mut config = self.common.read_config()?;
+        config.auth = false;
```

Comment:
What do these lines accomplish?

Reply:
So a lot of the recent work in the warg cli is dealing with the fact that in order to figure out whether a credential in your keyring even exists, there's a password prompt.

Basically this bool, along with the list of signing key names that was added in prior prs is stored in the config, and later the warg client is able to determine via config that keyring access shouldn't be attempted, so that it can be more intentional about keyring access when we know that it will fail.

Here specifically logging out removes your auth token from your keyring, so the config is updated for the reason described above.  I do realize that I'm missing a `config.write_to_file` here, so I can add that along with the other recommended changes.

Reply:
I figured we were failing to save the config back out; thanks for the confirmation!